client/src/App.tsx

```diff
@@ -1,13 +1,23 @@
-import Home from "./pages/Home";
-import {Route, Routes} from "react-router-dom";
+import Home from './pages/Home'
+import Login from './pages/Login'
+import { Route, Routes } from 'react-router-dom'
+import { ApolloClient, InMemoryCache, ApolloProvider, HttpLink, from } from '@apollo/client'
+
+const link = from([new HttpLink({ uri: 'http://localhost:3000/graphql' })])
+
+const client = new ApolloClient({
+  cache: new InMemoryCache(),
+  link,
+})
```

Comment:
These are configurations, hence they should be placed in a more central place for all configurations.
Create a new tsx file called `config.tsx` then add the configuration there.